docs/user-guide/cli-updatingcli.md

```diff
@@ -29,7 +33,15 @@ If you have an `@lts-incremental` version of Zowe CLI (Zowe v1.0.x - v1.8.x), yo
 
 4. Install the most recent `@zowe-v1-lts` version CLI and optional plug-ins. For more information, see [Installing CLI](./cli-installcli.md).
 
-5. **(Optional)** Reestablish your user profiles. Move the profile configuration files that you saved in Step 1 into the `~/.zowe/profiles` or `%homepath%\.zowe\profiles` folder on your computer.
+5. **(Optional)** If you deleted your profiles in Step 1, recreate the profiles that you need manually.
+
+6. **(Optional)** If you copied your profiles to a local directory in Step 1, follow these steps:
+
+   **a.**  Move the profile configuration files that you saved in Step 1 to the `~/.zowe/profiles` or `%homepath%\.zowe\profiles` folder on your computer.
```

Comment:
"you saved in Step 1 to" -> "you saved in Step 1 back to"
Feel free to ignore this comment 😄 I think adding the word "back" makes the connection to Step 1 clearer.